src/pages/ConsumerAgreementsListPage/components/ConsumerAgreementsTableRow.tsx

```diff
@@ -36,10 +37,29 @@ export const ConsumerAgreementsTableRow: React.FC<{ agreement: AgreementListEntr
     queryClient.prefetchQuery(AgreementQueries.getSingle(agreement.id))
   }
 
+  const isDelegator = Boolean(
+    agreement.delegation && agreement.delegation?.delegator.id === jwt?.organizationId
+  )
+  const isDelegate = Boolean(
+    agreement.delegation && agreement.delegation?.delegate.id === jwt?.organizationId
+  )
+
+  const isDelegated = isDelegate || isDelegator
```

Comment:
Here seems like  that who is watching this table`isDelegated` (while organizationId could be 'delegato' or 'delegante'). 
```suggestion
  const isADelegationFlowActive = isDelegate || isDelegator
```